plugins/mac-virtualcam/src/camera-extension/OBSCameraDeviceSource.swift

```diff
@@ -0,0 +1,288 @@
+//
+//  OBSCameraDeviceSource.swift
+//  mac-camera-extension
+//
+//  Created by Sebastian Beckmann on 30.09.22.
+//  Changed by Patrick Heyer on 16.10.22.
+//
+
+import AppKit
+import CoreMediaIO
+import Foundation
+import IOKit.audio
+import os.log
+
+let OBSCameraFrameRate: Int = 60
```

Comment:
Is the frame rate locked to 60 FPS?

Reply:
Correct, the device (as exposed to other applications) has a single resolution/framerate configuration - the incoming pixel buffers are converted automatically.